plugins/Dashboard/angularjs/dashboard/dashboard.directive.js

```diff
@@ -38,15 +38,18 @@
                     name: dashboard ? dashboard.name : ''
                 });
 
-            $('#columnPreview').find('>div').each(function () {
+            var divElements = $('#columnPreview').find('>div');
+
+            divElements.each(function () {
                 var width = [];
                 $('div', this).each(function () {
                     width.push(this.className.replace(/width-/, ''));
                 });
                 $(this).attr('layout', width.join('-'));
             });
 
-            $('#columnPreview').find('>div').on('click', function () {
+            divElements.off('click.renderDashboard');
+            divElements.on('click.renderDashboard', function () {
                 $('#columnPreview').find('>div').removeClass('choosen');
```

Comment:
Could this also be replaced with `divElements`?

As long as the events are always rebound when a new element is added there should be no reason not to.

Reply:
yes. correct. changed it